web/less/room.less

```diff
@@ -1348,3 +1352,22 @@ emoji-picker {
     position: relative;
     bottom: 345px;
 }
+
+.invite-btn {
+  @media @smartphonesPortrait {
+    display: none!important;
+  }
+}
+
+.duration {
+      @media @smartphonesPortrait {
+        display: inline-flex!important;
+        font-size: 9px;
+      }
+
+      @media @smartphonesLandscape{
+        display: inline-flex!important;
+        font-size: 9px;
+      }
```

Comment:
Change indenting to two spaces, please.

Reply:
In the Chrome simulator for iOS, the alignment of the #viewRecordings element is off:

![image](https://user-images.githubusercontent.com/726379/97263306-edd16100-17df-11eb-89ca-dd7762827a3b.png)

I'm having trouble seeing this on my iPhone, because of ngrok throttling limitations.

Reply:
On my iPhone Xs (in portrait and landscape mode) I do not see the #viewRecordings element.

Reply:
the managerecordings should be hidden for mobile. Please reload the chrome simulator in mobile view and it should be hidden

Reply:
updated in new commit

Reply:
OK, but I still see this behavior when I shrink a Chrome desktop window smaller.

Reply:
latest commit should not have this distorted view. but it is known fact that header is being cropped

Reply:
Maybe we should just hide the viewRecordings element (the righthand side of the button) in landscape portrait. I think the design looks a little cramped.